Kitodo/src/main/webapp/WEB-INF/templates/includes/processFromTemplate/import.xhtml

```diff
@@ -18,48 +18,64 @@
-                             required="#{not empty param['editForm:processFromTemplateTabView:performCatalogSearch']}"
-                             immediate="true"
-                             value="#{CreateProcessForm.importTab.hitModel.selectedCatalog}">
-                <f:selectItem itemValue="#{null}" itemLabel="-- #{msgs.selectCatalog} --" noSelectionOption="true"/>
-                <f:selectItems value="#{CreateProcessForm.importTab.catalogs}" var="step" itemLabel="#{step}"
-                               itemValue="#{step}"/>
-                <p:ajax update="editForm:processFromTemplateTabView:catalogSearchField,
-                                editForm:processFromTemplateTabView:catalogSearchTerm,
-                                editForm:processFromTemplateTabView:catalogSearchButton"/>
-            </p:selectOneMenu>
-        </div>
-        <h:panelGroup layout="block"
-                      id="catalogSearchField">
-            <p:outputLabel for="fieldSelectMenu"
-                           value="#{msgs.field}"
-                           rendered="#{CreateProcessForm.importTab.hitModel.selectedCatalog ne null}"/>
-            <p:selectOneMenu id="fieldSelectMenu"
-                             rendered="#{CreateProcessForm.importTab.hitModel.selectedCatalog ne null}"
-                             required="#{not empty param['editForm:processFromTemplateTabView:performCatalogSearch']}"
-                             value="#{CreateProcessForm.importTab.hitModel.selectedField}">
-                <f:selectItem itemValue="#{null}"
-                              itemLabel="-- #{msgs.selectSearchField} --"
-                              noSelectionOption="true"/>
-                <f:selectItems value="#{CreateProcessForm.importTab.searchFields}" var="field"/>
-            </p:selectOneMenu>
-        </h:panelGroup>
-        <h:panelGroup layout="block"
-                      id="catalogSearchTerm">
-            <p:outputLabel for="searchTerm"
+    <p:panelGrid layout="grid" columns="2" cellpadding="10">
+        <p:row>
+            <div>
+                <p:outputLabel for="catalogueSelectMenu"
+                               value="#{msgs['newProcess.catalogueSearch.catalogue']}"/>
+                <p:selectOneMenu id="catalogueSelectMenu"
+                                 required="#{not empty param['editForm:processFromTemplateTabView:performCatalogSearch']}"
+                                 immediate="true"
+                                 value="#{CreateProcessForm.importTab.hitModel.selectedCatalog}">
+                    <f:selectItem itemValue="#{null}" itemLabel="-- #{msgs.selectCatalog} --" noSelectionOption="true"/>
+                    <f:selectItems value="#{CreateProcessForm.importTab.catalogs}" var="step" itemLabel="#{step}"
+                                   itemValue="#{step}"/>
+                    <p:ajax update="editForm:processFromTemplateTabView:fieldSelectMenu,
```

Comment:
This would update the whole `tabView` including all five tabs and their content. I just want to update the three input components on the `import` tab, so I think the update should the stay the way it is.